test/README

```diff
@@ -219,3 +219,32 @@ Still, within a long test, try to have independent sections, where
 each section returns the machine to more or less the state that it was
 in before the section.  This makes it easier to run these sections
 ad-hoc when doing incremental development.
+
+## Running Selenium tests locally
+
+Selenium tests run as a part of GiHub PR CI.
+To run selenium tests locally you will need to do steps described below.
```

Comment:
This paragraph shouldn't be here IMHO. *All* tests are running in GitHub PRs, so that's not specific to the selenium tests. And the second sentence just repeats the heading.